recipes/libtmux/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "libtmux" %}
+{% set version = "0.36.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 12b5554b3a19d663d2a04f30b87fb063bd6456463a3ef6c6445a721fd7f7569a
+
+build:
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
+  number: 0
+  skip: true  # [win]
```

Comment:
My guess is that these packages can be OS-noarch. See https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies

Reply:
>  is it OK to change that file in the PR on staged-recipe?

You have to add one in the recipe directory, that one will be ported to the feedstock.